support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -262,6 +262,99 @@ export function Checkout() {
 	const [accountHolderConfirmation, setAccountHolderConfirmation] =
 		useState(false);
 
+	const formOnSubmit = async (formData: FormData) => {
+		/**
+		 * The validation for this is currently happening on the client side form validation
+		 * So we'll assume strings are not null.
+		 * see: https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation
+		 */
+		const data = {
+			firstName: formData.get('firstName') as string,
```

Comment:
Why do we need all these `as string`s?

Reply:
Oh it's because they could be null right?

Reply:
They could be null when using the `get` API - but we know they're not. 

There's probably a much nicer way to do this - I am waiting for the validation of this piece of work to help elucidate that.